src/main/java/de/tum/in/www1/artemis/service/QuizExerciseService.java

```diff
@@ -323,7 +323,7 @@ public void validateQuizExerciseFiles(QuizExercise quizExercise, @Nonnull List<M
         Set<String> exerciseFileNames = getAllPathsFromDragAndDropQuestionsOfExercise(quizExercise);
         Set<String> newFileNames = isCreate ? exerciseFileNames : exerciseFileNames.stream().filter(fileNameOrUri -> {
             try {
-                return !Files.exists(filePathService.actualPathForPublicPathOrThrow(URI.create(fileNameOrUri)));
+                return !Files.exists(FilePathService.actualPathForPublicPathOrThrow(URI.create(fileNameOrUri)));
```

Comment:
same here. (I won't comment on all affected classes now, please double-check that)